helm/leanix-k8s-connector/templates/clusterrole.yaml

```diff
@@ -7,42 +7,12 @@ metadata:
 {{ include "leanix-k8s-connector.labels" . | indent 4 }}
 rules:
 - apiGroups: [""]
-  resources:
-  - nodes
-  - persistentvolumes
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups: ["apiextensions.k8s.io"]
-  resources:
-  - customresourcedefinitions
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups: ["policy"]
-  resources:
-  - podsecuritypolicies
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups: ["rbac.authorization.k8s.io"]
-  resources:
-  - roles
-  - clusterroles
-  - rolebindings
-  - clusterrolebindings
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups: ["storage.k8s.io"]
-  resources:
-  - storageclasses
-  verbs:
-  - get
-  - list
-  - watch
-{{- end -}}
\ No newline at end of file
+  resources: ["nodes","persistentvolume","persistentvolumeclaim","namespaces","services"]
```

Comment:
For my understanding...do adding these resources in the list make permissions more fine-grained? Doesn't the node have an "overview" of all these resources?

Reply:
@daniellesarmiento From the API perspective https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#node-v1-core for `[get,list,watch]` it does not consider these resources. However even if we use `delete` the API server takes care of it. Thats the abstraction of  REST API operation w.r.t the actions on the resources.